autodock-vina/autodock.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+import qarnot
+import sys
+
+# Create a connection
+conn = qarnot.Connection('qarnot.conf')
```

Comment:
use client_token='MY_SECRET_TOKEN instead of qarnot.conf.  Our external doc don't use qarnot.conf in order to not mislead our users